src/java.base/share/classes/jdk/internal/misc/ThreadTracker.java

```diff
@@ -39,15 +39,25 @@ public class ThreadTracker {
      * The hashCode/equals methods do not invoke the Thread hashCode/equals method
      * as they may run arbitrary code and/or leak references to Thread objects.
      */
-    private record ThreadRef(Thread thread) {
+    private class ThreadRef {
+        private final Thread thread;
+
+        ThreadRef(Thread thread) {
+            this.thread = thread;
+        }
+
+        Thread thread() {
+            return this.thread;
+        }
+
         @Override
```

Comment:
Java 11 doe not have `record` yet, so we convert this `record` type as `inner class`.